apps/ui-library/styles/globals.css

```
@@ -1,82 +1,120 @@
-@import 'tailwindcss';
-
-@import './../../../packages/ui/build/css/source/global.css';
-@import './../../../packages/ui/build/css/themes/dark.css';
+@import 'config/tailwind.config.css';
 @import './../../../packages/ui/build/css/themes/classic-dark.css';
-@import './../../../packages/ui/build/css/themes/light.css';
 
-@config '../tailwind.config.js';
+@source '../app/**/*.{js,ts,jsx,tsx}';
```

Stylelint (17.10.0)

[error] Unexpected unknown at-rule "@source" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)